draft-fett-oauth-sd-jwt-vc-types.md

```diff
@@ -63,112 +63,137 @@ In an SD-JWT VC, `vct` can be used to express a type. Metadata can be retrieved
+Here, the type is `https://betelgeuse.example.com/education_credential`. The
 metadata is retrieved from the URL
-`https://betelgeuse.example.com/.well-known/vct/identity_credential`.
+`https://betelgeuse.example.com/.well-known/vct/education_credential`.
 
 The following is an example for a metadata document:
 
 ```json
 {
-  "vct": "https://betelgeuse.example.com/identity_credential",
-	"name": "Betelgeuse Education Credential - Preliminary Version",
-	"description": "This is our development version of the education credential. Don't panic.",
-	"extends": "https://galaxy.example.com/galactic-education-credential-0.9",
-	"display": [
-		{
-			"en-US": {
-				"name": "University Credential",
-                "rendering": {
-                    "simple": {
-                        "logo": {
-                            "uri": "https://exampleuniversity.com/public/logo.png",
-                            "uri#integrity": "sha256-LmXfh-9cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1V",
-                            "alt_text": "a square logo of a university"
-                        },
-                        "background_color": "#12107c",
-                        "text_color": "#FFFFFF",
-                    },
-                    "svg_templates": [
-                        {
-                            "uri": "https://exampleuniversity.com/public/credential-english.svg",
-                            "uri#integrity": "sha256-8cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1VLmXfh-9c",
-                            "properties": {
-                                "orientation": "landscape",
-                                "color_scheme": "light",
-                                "contrast": "high",
-                            }
-                        }
-                    ]
-                }
-			}
-		}
-	],
-	"claims":{
-        "/name": {
-            "display": {
-                "de-DE": {
-                    "label": "Vor- und Nachname",
-                    "description": "Der Name des Studenten"
-                },
-                "en-US": {
-                    "label": "Name",
-                    "description": "The name of the student"
-                }
+  "vct":"https://betelgeuse.example.com/education_credential",
+  "name":"Betelgeuse Education Credential - Preliminary Version",
+  "description":"This is our development version of the education credential. Don't panic.",
+  "extends":"https://galaxy.example.com/galactic-education-credential-0.9",
+  "extends#integrity":"sha256-9cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1VLmXfh-WRL5",
+  "display":[
+    {
+      "en-US":{
+        "name":"Betelgeuse Education Credential",
+        "rendering":{
+          "simple":{
+            "logo":{
+              "uri":"https://betelgeuse.example.com/public/education-logo.png",
+              "uri#integrity":"sha256-LmXfh-9cLlJNXN-TsMk-PmKjZ5t0WRL5ca_xGgX3c1V",
+              "alt_text":"a square logo of a university"
```

Comment:
Possibly isn't quite aligned, perhaps:

```suggestion
              "alt_text":"Betelgeuse Ministry of Education logo"
```

I can't make a suggestion for the German translation, sorry :-)